src/dosbox.cpp

```diff
@@ -958,16 +963,25 @@ void DOSBOX_Init() {
 	Pmulti_remain->GetSection()->Add_string("parameters", when_idle, "");
 	Pmulti_remain->Set_help(
 	        "set type of device connected to com port.\n"
-	        "Can be disabled, dummy, modem, nullmodem, directserial.\n"
+	        "Can be disabled, dummy, serialmouse, modem, nullmodem, directserial.\n"
 	        "Additional parameters must be in the same line in the form of\n"
 	        "parameter:value. Parameter for all types is irq (optional).\n"
```

Comment:
I don't think that's true anymore: "Parameter for all types is irq (optional)"

Reply:
Probably it is still true, it is handled by the serialport.cpp - not by concrete device emulators.

 I just did a small test, added irq:6 parameter to serialmouse - and it worked, CtMouse did not see mouse events on serial ports anymore until I asked him to use IRQ 6 too (/S16 command line parameter for COM1 IRQ6).